.github/ct-install.yaml

```diff
@@ -16,6 +16,7 @@ chart-dirs:
 
 chart-repos:
   - newrelic-helm-charts=https://newrelic.github.io/helm-charts
+  - newrelic-cdn-helm-charts=https://helm-charts.newrelic.com
```

Comment:
i didn't get why this is need, out of curiosity what is `newrelic-cdn-helm-charts` ?